winch/codegen/src/isa/x64/masm.rs

```diff
@@ -368,7 +342,14 @@ impl Masm for MacroAssembler {
                         .xmm_vpbroadcast_mr(&src, dst, size, UNTRUSTED_FLAGS);
                 }
             }
+            LoadKind::VectorLane(LaneSelector { lane, size }) => {
+                let byte_tmp = regs::scratch();
```

Comment:
The failures that CI are related to the `vpinsr*`instruction, which are emitted here. We need to add `self.ensure_has_avx()?` to get CI green.

Reply:
I setup the action on my fork, all tests pass now. worry for the back and forth :sweat: